packages/fxa-content-server/app/styles/modules/_password-row.scss

```diff
@@ -1,371 +0,0 @@
-    text-align: center;
-
-    br {
-      margin-bottom: 20px;
-    }
-
-    html[dir='ltr'] & {
-      background-position: 12px 16px;
-      padding-left: 35px;
-    }
-
-    html[dir='rtl'] & {
-      background-position: calc(100% - 12px) 16px;
-      padding-right: 35px;
-    }
-  }
-
-  .show-password {
-    opacity: 0;
-    position: absolute;
-    width: 1px;
-  }
-
-  /**
-   * The addition of right, top, border-radius, and background-color are
-   * all to fix a bug in firefox 29 where the right padding of the password
-   * field is ignored and password text flows under the show/hide button. When
-   * Firefox is fixed, top and right can be set to 0, height set to 44px,
-   * border-radius and background-color removed.
-   *
-   * See https://bugzilla.mozilla.org/show_bug.cgi?id=965237
-   */
-
-  .show-password-label {
-    background-color: $input-background-color-default;
-    border: 0;
-    cursor: pointer;
-    height: $input-height - 2px;
-    position: absolute;
-    top: 1px;
-    width: 55px;
-    z-index: 3;
-
-    svg {
-      height: 100%;
-      padding: 5px;
-      pointer-events: none;
-      width: 100%;
-
-      @include respond-to('small') {
-        padding: 0;
-      }
-    }
-
-    @include respond-to('small') {
-      height: $input-height-small - 2;
-    }
-
-    html[dir='ltr'] & {
-      border-radius: 0 $input-inner-border-radius $input-inner-border-radius 0;
-      right: 1px;
-    }
-
-    html[dir='rtl'] & {
-      border-radius: $input-inner-border-radius 0 0 $input-inner-border-radius;
-      left: 1px;
-    }
-  }
-
-  .password:focus ~ .show-password-label {
-    border-color: $input-border-color-focus;
-  }
-
-  .password.empty ~ .show-password,
-  .password.empty ~ .show-password-label {
-    display: none;
-  }
-
-  .input-help-balloon {
-    background-color: $color-white;
-    border: 1px solid $marketing-border-color;
-    border-radius: $small-border-radius;
-    box-shadow: 0 2px 8px rgba($grey-90, 0.1);
-    color: $color-black !important; // override default tip styles
-    padding: 15px;
-    position: absolute;
-    text-align: left;
-    width: 250px;
-    z-index: 6; // one more than the tooltip
-
-    span {
-      color: $grey-90;
-      display: block;
-    }
-  }
-
-  .input-help-balloon::before {
-    background-color: #fff;
-    border: solid #d7d7db;
-    content: '.';
-    height: 16px;
-    pointer-events: none;
-    position: absolute;
-    text-indent: -999px;
-    transform: rotate(45deg);
-    white-space: nowrap;
-    width: 16px;
-    // The z-index must be -1 or else the caret is displayed on top of the tooltip text
-    z-index: -1;
-  }
-
-  .password-strength-balloon {
-    font-size: 12px;
-    font-weight: 400;
-    line-height: 1.5;
-    opacity: 1;
-    padding: 16px;
-    z-index: 5;
-
-    // The characters in Arabic look way too small at 14px and are difficult to read. Bump them up by 1
-    // See #6556
-    html[lang^='ar'] & {
-      font-size: 15px;
-    }
```

Comment:
Something to note for removing this change - I'm going to file an issue in our [Modernize Localization](https://mozilla-hub.atlassian.net/browse/FXA-999) epic to review our font-sizing at our different locales, maybe that's something the l10n team can help us with, especially Arabic since we originally overrode styles here to make the font `15px`. I can't help but think if that was a problem here, it's going to be a problem everywhere we're using a small font-size. On the other hand, users should be able to increase text size themselves if they need to, _but_, we're using `px` instead of `rem` for our font-sizes since we wanted more control over the font-sizing so it might be good to review.